tests/app/views/test_briefs.py

```diff
@@ -596,6 +598,24 @@ def test_list_briefs_by_lot_and_status(self):
         assert res.status_code == 200
         assert len(data['briefs']) == 3, data['briefs']
 
+    def test_list_briefs_by_human_readable(self):
+        today = datetime.utcnow()
+        self.setup_dummy_briefs(1, data={'requirementsLength': '2 weeks'}, status=None, title='First',
+                                published_at=(today + timedelta(days=-40)))
+        self.setup_dummy_briefs(1, data={'requirementsLength': '2 weeks'}, status=None, title='Second',
+                                published_at=(today + timedelta(days=-9)), brief_start=2)
+        self.setup_dummy_briefs(1, data={'requirementsLength': '1 week'}, status=None, title='Third',
+                                published_at=(today + timedelta(days=-8)), brief_start=3)
+        self.setup_dummy_briefs(1, data={'requirementsLength': '1 week'}, status=None,  title='Fourth',
+                                published_at=(today + timedelta(days=-2)), brief_start=4)
+
+        res = self.client.get('/briefs?human=True')
+        data = json.loads(res.get_data(as_text=True))
+        titles = map(lambda brief: brief['title'], data['briefs'])
```

Comment:
This is going to break in Python3 😱 

It seems like [you want to wrap a `list()` around it](http://stackoverflow.com/questions/1303347/getting-a-map-to-return-a-list-in-python-3-x).